internal/services/gateway/api/remotesource.go

```diff
@@ -238,3 +238,30 @@ func (h *DeleteRemoteSourceHandler) ServeHTTP(w http.ResponseWriter, r *http.Req
 		h.log.Err(err).Send()
 	}
 }
+
+type RemoteSourceUserHandler struct {
+	log zerolog.Logger
+	ah  *action.ActionHandler
+}
+
+func NewRemoteSourceUserHandler(log zerolog.Logger, ah *action.ActionHandler) *RemoteSourceUserHandler {
+	return &RemoteSourceUserHandler{log: log, ah: ah}
+}
+
+func (h *RemoteSourceUserHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	vars := mux.Vars(r)
+	rsRef := vars["remotesourceref"]
+	remoteUserId := vars["remoteuserid"]
+
+	user, err := h.ah.GetUserByLinkedAccountRemoteUserAndSource(ctx, remoteUserId, rsRef)
+	if util.HTTPError(w, err) {
+		h.log.Err(err).Send()
+		return
+	}
+
+	res := createUserResponse(user)
```

Comment:
Is this enough, or should be better to return a `gwapitypes.PrivateUserResponse` providing also the linked accounts?